assets/helpers/url.js

```diff
@@ -35,6 +35,10 @@ const getQueryParameter = (paramName: string, defaultValue?: string): ?string =>
 
 };
 
+const getQueryParams = (excluded: string[]): [string, string][] => Array
+  .from(new URLSearchParams(window.location.search))
```

Comment:
If I am reading [this table correctly](https://polyfill.io/v2/docs/features/#Array_from) `Array.from` would not be supported in IE and it is not available in polyfill. Did you test that it works in IE?

Reply:
I've checked and it works fine in IE with the addition of `.entries()` to the end of the `new UrlSearchParams()`